CONTRIBUTING.md

```diff
@@ -2,9 +2,9 @@
 
 ## General
 
-* Contributions of all kinds (issues, ideas, proposals), not just code, are highly appreciated.
-* Pull requests are welcome with the understanding that major changes will be carefully evaluated and discussed, and may not always be accepted. Starting with a discussion is always best!
-* All contributions including documentation, filenames and discussions should be written in the English language.
+- Contributions of all kinds (issues, ideas, proposals), not just code, are highly appreciated.
+- Pull requests are welcome with the understanding that major changes will be carefully evaluated and discussed, and may not always be accepted. Starting with a discussion is always best!
+- All contributions including documentation, filenames and discussions should be written in the English language.
```

Comment:
Since this does not have to do with adding Windows support we should probably edit it back to how it was as we add instructions on how to run the app on Windows.

Reply:
As I mentioned, this wasn't a change that I made and any attempts to restore it are overridden during the precommit process. I'll look into what's going on there as it might be some setting that needs to be disabled. Seems like it's enforcing some style guide on Md files.